docs/usage/rust.md

```diff
@@ -11,14 +11,19 @@ Renovate supports upgrading dependencies in `Cargo.toml` files and their accompa
 
 1. Renovate searches in each repository for any `Cargo.toml` files
 1. Renovate extracts existing dependencies from `[dependencies]`, `[dev-dependencies]`, `[build-dependencies]` and `[workspace.dependencies]`
-1. Renovate tries to find and parse a `.cargo/config.toml` file to discover index URLs for private registries
+1. Renovate looks up Cargo configuration to discover index URLs for private registries.
 1. Renovate resolves the dependency's version using the crates.io API or by cloning the index URL
 1. If Renovate finds an update, Renovate will use `cargo update` to update both `Cargo.toml` and `Cargo.lock`
 
 ## Enabling Rust Modules Updating
 
 Renovate updates Rust crates by default.
 
+## Cargo configuration and private registry discovery
+
+Renovate can determine private registry URLs from a `.cargo/config.toml` (or the legacy `.cargo/config`) Cargo configuration file.
```

Comment:
```suggestion
Renovate can find private registry URLs in these Cargo configuration files:

- `.cargo/config.toml`
- `.cargo/config` (legacy)
```

Changes:

- Put config files in a list
- Replace `determine` with the simple word `find`
- Rewrite the sentence a little bit